src/main/java/com/mattworzala/canary/server/instance/block/LecternHandler.java

```diff
@@ -53,11 +58,30 @@ public boolean onInteract(@NotNull Interaction interaction) {
                 .append(Component.text(testFailure))
                 .build();
 
+        TextComponent errorText;
+        String hasteURL = testHasteUrl;
+        try {
+            if (testHasteUrl == null) {
+                Hastebin hastebin = new Hastebin();
+                hasteURL = hastebin.post(testStacktrace, false);
+                interaction.getInstance().setBlock(interaction.getBlockPosition(), block.withTag(Tags.TestHasteUrl, hasteURL));
+            }
+
+            TextComponent urlComponent = Component.text(hasteURL)
```

Comment:
The text should be something simple like "View on Hastebin" rather than the url which doesn't tell much.

Reply:
Yeah